recurly/__init__.py

```diff
@@ -521,6 +522,67 @@ class Delivery(Resource):
         'personal_message',
     )
 
+class DunningCampaign(Resource):
+
+    """A dunning campaign available on the site"""
+
+    member_path = 'dunning_campaigns/%s'
+    collection_path = 'dunning_campaigns'
+
+    nodename = 'dunning_campaign'
+
+    attributes = (
+      'id',
+      'code',
+      'name',
+      'description',
+      'default_campaign',
+      'dunning_cycles',
+      'created_at'
+      'updated_at',
+      'deleted_at',
+      'plans',
+    )
+
+    def bulk_update(self, id, plan_codes):
+
+        """Update each plan's `dunning_campaign_id`."""
+
+        url = urljoin(base_uri(), self.member_path % (id) + '/bulk_update')
+
+        elem = ElementTreeBuilder.Element(self.nodename)
+        plan_codes_elem = ElementTreeBuilder.Element('plan_codes')
+        elem.append(plan_codes_elem)
+
+        for plan_code in plan_codes:
+            plan_codes_elem.append(Resource.element_for_value('plan_code', plan_code))
+
+        body = ElementTree.tostring(elem, encoding='UTF-8')
+
+        self.http_request(url, 'PUT', body, { 'content-type':
+            'application/xml; charset=utf-8' })
+
+class DunningCycle(Resource):
+
+    """A dunning cycle associated to a dunning campaign."""
+
+    nodename = 'dunning_cycle'
+
+    attributes = (
+        'type',
+        'applies_to_manual',
```

Comment:
Should this be `'applies_to_manual_trial'`? Currently, it raises an attribute error.
```suggestion
        'applies_to_manual_trial',
```